Tests/NetworkingTests/Mocks/MockFileDataWriter.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  FileDataWriterSpy.swift
+//
+//
+//  Created by Jan Kodeš on 24.01.2024.
+//
+
+import Foundation
+import Networking
+
+/// A test mock class for `FileDataWriting`.
+/// It writes into a file but let's us react when it's finished.
+class MockFileDataWriter: FileDataWriting {
```

Comment:
We have MockFileManager final (I assume all mock classes) can we have it final as well?